### PR TITLE
update gateway config

### PR DESCRIPTION
**Pull request by jeanmi151**
update gateway config with default configuration present here : https://github.com/georchestra/georchestra-gateway/blob/main/gateway/src/main/resources/application.yml#L92

### CODE REVIEW

**Review by pmauduit — APPROVED** [review]

File: gateway/gateway.yaml [hunk 1]
```diff
@@ -48,30 +48,30 @@ georchestra:
...
           anonymous: true
         - intercept-url:
-          - /console/private/.*
-          - /console/manager/.*
-          - /console/.*/emails
-          - /console/.*/sendEmail # /console/sendEmail features are reserved to SUPERUSER & delegated admins
-          - /console/.*/emailTemplates
+          - /console/private/**
+          - /console/manager/**
+          - /console/*/emails
```
Comment (fvanderbiest) [inline gateway/gateway.yaml:65]: missing `**` here and there

**Comment by fvanderbiest** [comment]
That was too fast for me :-)

**Comment by jeanmi151** [comment]
oh sorry ! reopening and making to modifications

File: gateway/gateway.yaml (hunk 1 above)
Comment (pmauduit) [reply gateway/gateway.yaml:65]: I did not get @fvanderbiest's comments: It is not possible to have double wildcard if not at the end of the entry, else spring won't start (can't recall exactly the reason).

Comment (fvanderbiest) [reply gateway/gateway.yaml:65]: Indeed. 
I was fooled by your commits on the MEL datadir (did not see the last one)

**Pull request merged**